From: Vlastimil Babka <vbabka@suse.cz>
Subject: mm-use-stack_depot_early_init-for-kmemleak-fix
Date: Thu, 19 Jan 2023 11:32:36 +0100

The call to stack_depot_init() becomes no-op after this patch so it's not
a bug to leave it there, but it's just misleading now.

Signed-off-by: Vlastimil Babka <vbabka@suse.cz>
Cc: Catalin Marinas <catalin.marinas@arm.com>
Cc: ke.wang <ke.wang@unisoc.com>
Cc: Mike Rapoport (IBM) <rppt@kernel.org>
Cc: Mirsad Goran Todorovac <mirsad.todorovac@alu.unizg.hr>
Cc: Nathan Chancellor <nathan@kernel.org>
Cc: Peter Zijlstra (Intel) <peterz@infradead.org>
Cc: Zhaoyang Huang <zhaoyang.huang@unisoc.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/mm/kmemleak.c~mm-use-stack_depot_early_init-for-kmemleak-fix
+++ a/mm/kmemleak.c
@@ -2094,7 +2094,6 @@ void __init kmemleak_init(void)
 	if (kmemleak_error)
 		return;
 
-	stack_depot_init();
 	jiffies_min_age = msecs_to_jiffies(MSECS_MIN_AGE);
 	jiffies_scan_wait = msecs_to_jiffies(SECS_SCAN_WAIT * 1000);
 
_
